src/log/logger_config.cpp

```diff
@@ -123,11 +125,23 @@ namespace fc {
                      ( "level_colors", c )
                  ) );
 
+      cfg.appenders.push_back(
+             appender_config( "deep-mind", "dmlog" )
+      );
+
       logger_config dlc;
       dlc.name = DEFAULT_LOGGER;
       dlc.level = log_level::info;
       dlc.appenders.push_back("stderr");
       cfg.loggers.push_back( dlc );
+
+      logger_config dmlc;
+      dmlc.name = "deep-mind";
+      dmlc.level = log_level::info;
+      dmlc.enabled = true;
+      dmlc.appenders.push_back("deep-mind");
+      cfg.loggers.push_back( dmlc );
+
```

Comment:
Don't add `"deep-mind'` as a default logger.

Reply:
We would really like to reduce the necessary setup for people to use `deep mind` instrumented `nodeos` process. We would largely prefer a single config that enables everything.

Is there anyway I could setup all this only if `deep-mind = true`? That would remove this code here and moving else where, where necessary changes would be applied only when `deep-mind = true` is activated specifically.

Reply:
You could go back to the old `initialize_logging` changes that augment the `logger_config` to add the `deep-mind` appender (using the now already registered `dmlog` appender type) and the `deep-mind` logger at `debug` level. The main difference from the previous iteration is that now `dmlog` appender is part of fc rather than a separate library within EOSIO

Reply:
Perfect.